client/components/SurrenderForm.js

```diff
@@ -0,0 +1,197 @@
+      const body = await response.json();
+      setPosted(true);
+    } catch (err) {
+      console.error(`Error in fetch: ${err.status} (${err.message})`);
+    }
+  }
+
+  const onSubmit = async event => {
+    event.preventDefault();
+    let errorsArray = checkForErrors();
+    if (errorsArray.length === 0) {
+      await addSurrenderInfo();
+      clearForm();
+      setErrors([]);
+    } else {
+      setErrors(errorsArray);
+    }
+  }
+
+  const message = posted ? "Your surrender request is in process." : "";
+  return (
+    <div>
+      <div>{errorContainer}</div>
+      <h2>Pet Surrender Application</h2>
+      <h3>{message}</h3>
+      <form onSubmit={onSubmit}>
+        <label htmlFor="name" >
+          Name:
+      <input
+            id="name"
+            type="text"
+            name="name"
+            value={surrenderData.name}
+            onChange={onChange} />
+        </label>
+        <label htmlFor="phone-number">
+          Phone Number:
+      <input
+            id="phone-number"
+            type="text"
+            name="phoneNumber"
+            value={surrenderData.phoneNumber}
+            onChange={onChange} />
+        </label>
+        <label htmlFor="email">
+          Email:
+      <input
+            id="email"
+            type="text"
+            name="email"
+            value={surrenderData.email}
+            onChange={onChange} />
+        </label>
+        <label htmlFor="petType">Pet Type
+    <select name="petType" id="petType" onChange={onChange} value={surrenderData.petType}>
+            <option value="Select Pet Type">Select Pet type</option>
+            <option value="dog">dog</option>
+            <option value="cat">cat</option>
+            <option value="hamster">hamster</option>
+            <option value="fish">fish</option>
+            <option value="turle">turtle</option>
+          </select>
+        </label>
+        <label htmlFor="petName">
+          Pet Name:
+      <input
+            id="petName"
+            type="text"
+            name="petName"
+            value={surrenderData.petName}
+            onChange={onChange} />
+        </label>
+        <label htmlFor="petAge">
+          Pet Age:
+      <input
+            id="petAge"
+            type="number"
+            name="petAge"
+            value={surrenderData.petAge}
+            onChange={onChange} />
+        </label>
+        <label htmlFor="petImage">
+          Pet Image:
+      <input
+            id="petImage"
+            type="text"
+            name="petImage"
+            value={surrenderData.petImage}
+            onChange={onChange} />
+        </label>
+        <label htmlFor="vaccinationStatus">Vaccination Status:
+      <select name="vaccinationStatus" id="vaccinationStatus" onChange={onChange} value={surrenderData.vaccinationStatus}>
+            <option value="Choose">Choose</option>
+            <option value="Yes">Yes</option>
+            <option value="No">No</option>
+          </select>
```

Comment:
Might just be how the indentation is coming across on GitHub, but take a look at the HTML structure here - things like the opening and closing `<select> tags should line up with one another